akka-bench-jmh/src/main/scala/akka/serialization/jackson/JacksonSerializationBench.scala

```diff
@@ -187,9 +187,13 @@ class JacksonSerializationBench {
   var serialization: Serialization = _
 
   @silent("immutable val") // JMH updates this via reflection
-  @Param(Array("jackson-json", "jackson-cbor")) // "java"
+  @Param(Array("jackson-json")) // "java"
```

Comment:
keep `"jackson-cbor"` in the comment, for convenience when we want to run it with cbor